From b2556eee0346f1a60e40daa369c7d0c040d78f71 Mon Sep 17 00:00:00 2001
From: Stefan Lippers-Hollmann <s.l-h@gmx.de>
Date: Wed, 5 Dec 2012 19:42:43 +0100
Subject: [PATCH] ds3000: force firmware download on default

Commit 034351ff6372c1a880ed4c5d333e430e9c83d467 ([media] ds3000: add
module parameter to force firmware upload) changes the default to no
longer load the firmware each time. The TeVii s480 needs this setting,
otherwise the firmware will never be uploaded upon a cold powerup.

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>
Cc: stable@vger.kernel.org # 3.7+
---
 drivers/media/dvb-frontends/ds3000.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/media/dvb-frontends/ds3000.c
+++ b/drivers/media/dvb-frontends/ds3000.c
@@ -30,7 +30,7 @@
 #include "ds3000.h"
 
 static int debug;
-static int force_fw_upload;
+static int force_fw_upload = 1;
 
 #define dprintk(args...) \
 	do { \
@@ -1310,7 +1310,7 @@ module_param(debug, int, 0644);
 MODULE_PARM_DESC(debug, "Activates frontend debugging (default:0)");
 
 module_param(force_fw_upload, int, 0644);
-MODULE_PARM_DESC(force_fw_upload, "Force firmware upload (default:0)");
+MODULE_PARM_DESC(force_fw_upload, "Force firmware upload (default:1)");
 
 MODULE_DESCRIPTION("DVB Frontend module for Montage Technology "
 			"DS3000/TS2020 hardware");
